tests/test_utils.py

```diff
@@ -116,9 +230,17 @@ def test_parse_activity_html_contents(mbz_path):
             parsed_html_content_strings.append(
                 html_elem.tostring().decode("utf-8")
             )
-
-    assert set([LESSON1_CONTENT1, LESSON1_CONTENT2, PAGE2_CONTENT]) == \
-        set(parsed_html_content_strings)
+    comparable_html = squish_list(parsed_html_content_strings)
```

Comment:
If you insert the space for the attributes in the constants, then I think you're just a `strip()` (on L231) away from having comparable strings. Something like:

```python
    parsed_html_content_strings = []
    for act in activities:
        for html_elem in act.html_elements():
            parsed_html_content_strings.append(
                html_elem.tostring().decode("utf-8").strip()
            )

    expected = [
        LESSON1_CONTENT1,
        LESSON1_CONTENT2,
        LESSON_ANSWER1,
        LESSON_ANSWER2,
        PAGE2_CONTENT,
        QUESTION1_CONTENT,
        QUESTION2_CONTENT,
        ANSWER1_CONTENT,
        ANSWER2_CONTENT
    ]
    assert set(expected) == set(parsed_html_content_strings)
```

The `strip()` is only necessary for a couple of elements in `lesson1_content` where the closing tag is on the next line introducing line breaks.